CruCentralCoast/Resources/SegmentedControlInNavBarView.swift

```diff
@@ -8,23 +8,31 @@
 
 import UIKit
 
-class SegmentedControlInNavBarView: UITableViewHeaderFooterView {
+class SegmentedControlInNavBarView: UIView {
     
     @IBOutlet weak var fakeBottomOfNavBar: UIView!
     
```

Comment:
It is generally cleaner code to put @IBOutlets next to each other at the top of the class